templates/repo/branch/list.tmpl

```diff
@@ -79,7 +79,7 @@
 											{{end}}
 										{{end}}
 									</td>
-									{{if and $.IsWriter (not $.IsMirror)}}
+									{{if and $.IsWriter (not $.IsMirror) (not $.Repository.IsArchived)}}
 										<td class="right aligned">
 										{{if .IsProtected}}
 											<i class="octicon octicon-shield"></i>
```

Comment:
The protected icon should be displayed even it's an archived repository?

Reply:
we can do this but then i have to change a lot more

Reply:
what do you think? @lunny

Reply:
or backport parts of PR https://github.com/go-gitea/gitea/pull/7461 
-> thie protection symbol is then in at front of the branch name